### ISSUE TITLE
Change Train_Manager.SaveModelHistory(tf.keras.callbacks.Callback) steps_per_epoch term

### ISSUE THREAD

**Issue by johnypark** [johnypark/mytflib#3]
For learning rate schedulers, there is a code that requires steps_per_epoch entry. The current code looks like this:
``` 
extra_logs[hyperparam] = float(value((epoch+1) * self.config_info["steps_per_epoch"])) ## here replace with self.params.get('steps')
```
Replace this part with a call that enables to get the steps_per_epoch term. See if self.params.get("steps") works.
